AppCenter/AppCenter/Internals/Model/CommonSchema/MSCSConstants.m

```diff
@@ -0,0 +1,62 @@
+#import "MSCSConstants.h"
+
+#pragma mark - MSCommonSchemaLog
+
+NSString *const kMSCSVer = @"ver";
+NSString *const kMSCSName = @"name";
+NSString *const kMSCSTime = @"time";
+NSString *const kMSCSPopSample = @"popSample";
+NSString *const kMSCSIKey = @"iKey";
+NSString *const kMSCSFlags = @"flags";
+NSString *const kMSCSCV = @"cV";
+NSString *const kMSCSData = @"data";
+NSString *const kMSCSExt = @"ext";
+
+#pragma mark - MSUserExtension
+
+NSString *const kMSUserLocale = @"locale";
+
+#pragma mark - MSLocExtension
+
+NSString *const kMSTimezone = @"tz";
```

Comment:
I think @guperrot changed it back to 'tz' since that's what the wiki said (https://osgwiki.com/wiki/Common_Schema_3.0#Loc_extension).  One Collector folks confirmed wiki will be our source for CS3.0.

Reply:
We decided to go with "tz" after asking this to schema experts.

Reply:
I have not changed it back yet, wanted to confirm. This is todo for Android.